src/python/bot/fuzzers/libFuzzer/strategy_selection.py

```diff
@@ -0,0 +1,139 @@
+
+from bot.fuzzers import engine_common
+from bot.fuzzers import strategy
+
+# Probability of using `-max_len` option. Not applicable if already declared in
+# .options file.
+RANDOM_MAX_LENGTH_PROBABILITY = 0.15
+
+# Probability of doing ML RNN mutations on the corpus in this run.
+CORPUS_MUTATION_ML_RNN_PROBABILITY = 0.50
+
+# Probability of doing radamsa mutations on the corpus in this run.
+CORPUS_MUTATION_RADAMSA_PROBABILITY = 0.15
+
+# Number of radamsa mutations.
+RADAMSA_MUTATIONS = 2000
+
+# Maximum number of seconds to run radamsa for.
+RADAMSA_TIMEOUT = 3
+
+# Probability of recommended dictionary usage.
+RECOMMENDED_DICTIONARY_PROBABILITY = 0.10
+
+# Probability of using `-use_value_profile=1` option.
+VALUE_PROFILE_PROBABILITY = 0.33
+
+FORK_PROBABILITY = 0.50
+
+MUTATOR_PLUGIN_PROBABILITY = 0.50
+
+
+def do_random_max_length():
+  """Return whether or not to do value profile."""
+  return engine_common.decide_with_probability(
+      engine_common.get_strategy_probability(
+          strategy.RANDOM_MAX_LENGTH_STRATEGY,
+          default=RANDOM_MAX_LENGTH_PROBABILITY))
+
+
+def use_generator():
+  """Return whether to use a generator or not."""
+  radamsa_p = engine_common.get_strategy_probability(
+      strategy.CORPUS_MUTATION_RADAMSA_STRATEGY,
+      default=CORPUS_MUTATION_RADAMSA_PROBABILITY)
+  ml_rnn_p = engine_common.get_strategy_probability(
+      strategy.CORPUS_MUTATION_ML_RNN_STRATEGY,
+      default=CORPUS_MUTATION_ML_RNN_PROBABILITY)
+  return engine_common.decide_with_probability(radamsa_p + ml_rnn_p)
+
+
+def do_radamsa_or_ml_rnn_generator():
+  """Return whether to use radamsa or ml rnn generator.
+
+  True for radamsa, false for ml rnn."""
+  radamsa_p = engine_common.get_strategy_probability(
+      strategy.CORPUS_MUTATION_RADAMSA_STRATEGY,
+      default=CORPUS_MUTATION_RADAMSA_PROBABILITY)
+  ml_rnn_p = engine_common.get_strategy_probability(
+      strategy.CORPUS_MUTATION_ML_RNN_STRATEGY,
+      default=CORPUS_MUTATION_ML_RNN_PROBABILITY)
+  return engine_common.decide_with_probability(
+      radamsa_p / (radamsa_p + ml_rnn_p))
+
+
+def do_recommended_dictionary():
+  """Return whether or not to use the recommended dictionary."""
+  return engine_common.decide_with_probability(
+      engine_common.get_strategy_probability(
+          strategy.RECOMMENDED_DICTIONARY_STRATEGY,
+          default=RECOMMENDED_DICTIONARY_PROBABILITY))
+
+
+def do_value_profile():
+  """Return whether or not to do value profile."""
+  return engine_common.decide_with_probability(
+      engine_common.get_strategy_probability(
+          strategy.VALUE_PROFILE_STRATEGY, default=VALUE_PROFILE_PROBABILITY))
+
+
+def do_fork():
+  """Return whether or not to do fork mode."""
+  return engine_common.decide_with_probability(
+      engine_common.get_strategy_probability(
+          strategy.FORK_STRATEGY, default=FORK_PROBABILITY))
+
+
+def do_mutator_plugin():
```

Comment:
These helper functions actually don't seem to useful to me now that they've been normalized. Could you just have a single function that you use for all of these simple cases that takes the strategy and probability as arguments?